mrp_operations_project/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==================================
+MRP Project Link (with operations)
+==================================
+
+This module links manufacturing operations with projects. It performs the
+following actions:
+
+* Auto-create tasks for work orders with assigned operators.
+* Add a tab where coding task works for the work order.
+* Add a link on task for the scheduled products for that work order.
+
+Installation
+============
+
+This modules is auto-installed when you install *mrp_operations_extension* and
+*mrp_project*.
+
+Usage
+=====
+
+In a manufacturing order (MO) that has work orders, and they have at least
```

Comment:
s/that has work orders, and they have at least/containing work orders with at least